utils.py

```diff
@@ -19,26 +20,27 @@ def configlogger():
 
 
 @event.listens_for(Engine, 'connect')
-def setsqlitepragma(dbapiconnection, connectionrecord):
+def set_sqlite_pragma(dbapiconnection, connectionrecord):
     """Activate sqlite pragma to enforce foreign keys integrity, in particular for cascade delete."""
     cursor = dbapiconnection.cursor()
     cursor.execute("PRAGMA foreign_keys=ON")
     cursor.close()
 
 
-def getfilename(path):
+def get_filename(path):
     """Extract file name from absolute path."""
     head, tail = ntpath.split(path)
     return tail or ntpath.basename(head)
 
 
-def getobjectattributes(object):
+def get_object_attributes(object):
     """Get attributes and their values from the instance of a class and returns them as a dictionary."""
     attributes = inspect.getmembers(object, lambda a: not(inspect.isroutine(a)))
     dictionary = {}
     for attribute in attributes:
-        if not attribute[0].startswith('_') and attribute[1] != [] and attribute[0] != 'metadata':
-            dictionary[attribute[0]] = str(attribute[1])
+        # Exclude object attributes belonging to sqlachemy and database classes
```

Comment:
not sure where this comes from, did you remove this commit ?
Might have been on my previous master branch.